tutorials/wikiqa/drmm.ipynb

```diff
@@ -235,163 +254,134 @@
+      "Validation: normalized_discounted_cumulative_gain@3(0.0): 0.5617858448784616 - normalized_discounted_cumulative_gain@5(0.0): 0.620981649738009 - mean_average_precision(0.0): 0.575758728526883\n",
       "Epoch 26/30\n",
-      "102/102 [==============================] - 7s 70ms/step - loss: 0.5968\n",
-      "Validation: normalized_discounted_cumulative_gain@3(0.0): 0.6165603731497318 - normalized_discounted_cumulative_gain@5(0.0): 0.6669051954046883 - mean_average_precision(0.0): 0.6259158352852249\n",
+      "102/102 [==============================] - 18s 180ms/step - loss: 0.2751\n",
+      "Validation: normalized_discounted_cumulative_gain@3(0.0): 0.5715964353403269 - normalized_discounted_cumulative_gain@5(0.0): 0.6274431994106784 - mean_average_precision(0.0): 0.5809745085309563\n",
       "Epoch 27/30\n",
-      "102/102 [==============================] - 8s 78ms/step - loss: 0.5910\n",
-      "Validation: normalized_discounted_cumulative_gain@3(0.0): 0.6087728632432321 - normalized_discounted_cumulative_gain@5(0.0): 0.6628481769034252 - mean_average_precision(0.0): 0.6202166294277911\n",
-      "Epoch 28/30\n",
-      "102/102 [==============================] - 6s 63ms/step - loss: 0.5887\n",
+      "102/102 [==============================] - 18s 179ms/step - loss: 0.2681\n",
+      "Validation: normalized_discounted_cumulative_gain@3(0.0): 0.5727264254869219 - normalized_discounted_cumulative_gain@5(0.0): 0.6251632237275019 - mean_average_precision(0.0): 0.5774814071969209\n",
       "Epoch 28/30\n",
-      "Validation: normalized_discounted_cumulative_gain@3(0.0): 0.6154908540530706 - normalized_discounted_cumulative_gain@5(0.0): 0.6629795666120435 - mean_average_precision(0.0): 0.624048508436513\n",
+      "102/102 [==============================] - 17s 164ms/step - loss: 0.2855\n",
+      "Validation: normalized_discounted_cumulative_gain@3(0.0): 0.572203475284389 - normalized_discounted_cumulative_gain@5(0.0): 0.6242293719131763 - mean_average_precision(0.0): 0.5801628256789166\n",
       "Epoch 29/30\n",
-      "102/102 [==============================] - 6s 63ms/step - loss: 0.5843\n",
-      "Validation: normalized_discounted_cumulative_gain@3(0.0): 0.6131571090299734 - normalized_discounted_cumulative_gain@5(0.0): 0.6612840152349587 - mean_average_precision(0.0): 0.620080578184145\n",
+      "102/102 [==============================] - 17s 166ms/step - loss: 0.2673\n",
+      "Validation: normalized_discounted_cumulative_gain@3(0.0): 0.5642022500077641 - normalized_discounted_cumulative_gain@5(0.0): 0.622231250656966 - mean_average_precision(0.0): 0.5736010477920581\n",
       "Epoch 30/30\n",
-      "101/102 [============================>.] - ETA: 0s - loss: 0.5823Validation: normalized_discounted_cumulative_gain@3(0.0): 0.6131571090299734 - normalized_discounted_cumulative_gain@5(0.0): 0.6612840152349587 - mean_average_precision(0.0): 0.620080578184145\n",
```

Comment:
The MAP score of DRMM on WikiQA is around 0.62? Maybe we can add DRMM_TKS to get better results.